CelesteNet.Client/Components/CelesteNetPlayerListComponent.cs

```diff
@@ -295,23 +323,31 @@ public CelesteNetPlayerListComponent(CelesteNetClientContext context, Game game)
 
                 // introducing gap after own channel
                 if (splitStartsAt > 0 && i == splitStartsAt) {
+                    // remove the blob spacing and replace it with y padding
+                    sizeAll.Y -= BlobSpacing * scale;
+                    sizeAll.Y += PaddingY * scale;
                     sizeUpper = sizeAll;
-                    blob.Dyn.Y += 30f * scale;
-                    sizeAll.Y += 30f * scale;
+                    blob.Dyn.Y += (2 * PaddingY + SplitGap) * scale;
```

Comment:
One thing I noticed while I'm in the process of reviewing this - if you move line 320 down to after 333 (set Dyn.Y after the if) you can complete remove line 330 because it does kinda the same thing as 328+332 but to Dyn.Y :)

Reply:
Did you mean line 330? Line 320 is the Blob variable declaration.